packages/blade/src/components/Modal/docs/ModalDocs.tsx

```diff
@@ -0,0 +1,207 @@
+import React from 'react';
+import { Playground } from './stories';
+import StoryPageWrapper from '~src/_helpers/storybook/StoryPageWrapper';
+import { Code, Heading, Text, Title } from '~components/Typography';
+import { Sandbox } from '~src/_helpers/storybook/Sandbox';
+import { Box } from '~components/Box';
+import { ArgsTable } from '~src/_helpers/storybook/ArgsTable';
+import { List, ListItem } from '~components/List';
+import BaseBox from '~components/Box/BaseBox';
+import { Link } from '~components/Link';
+
+const ModalDocs = (): JSX.Element => {
+  return (
+    <StoryPageWrapper
+      componentName="Modal"
+      componentDescription="Modals are often used to present information, notifications, or requests that require immediate attention or confirmation from the user"
+      imports=""
+      showStorybookControls={false}
+      figmaURL={{
+        paymentTheme:
+          'https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?type=design&node-id=39809-559333&t=j3ZK6zZSwP5CJ8Al-0',
+        bankingTheme:
+          'https://www.figma.com/file/sAdplk2uYnI2ILnDKUxycW/Blade---Banking-Dark?type=design&node-id=16459-684636&t=7KYEW7ENNS5GmLIO-0',
+      }}
+    >
+      <Box as="section">
+        <Title size="medium">Playground</Title>
+        <Sandbox editorHeight={400}>{Playground}</Sandbox>
+      </Box>
+
+      <Title size="medium" marginBottom="spacing.4">
+        ⚠️ Modal for mWeb and React Native
```

Comment:
Can probably also use the alert component

Reply:
Some alignment issue on fullWidth Alert. Raised it on slack, needs design input.